<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>475</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>475</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MiniCar上位机</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgFile.qrc">
    <normaloff>:/img/widget.ico</normaloff>:/img/widget.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QLabel" name="label_window_title">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/img/icon.png&quot; width=&quot;150&quot; height=&quot;60&quot;/&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;　　　MiniCar 上位机&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="BasicControl">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="title">
          <string>配置区域</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_ip">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>76</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="text">
               <string>IP 地址：</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_ip">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::InsertAtBottom</enum>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContents</enum>
              </property>
              <property name="duplicatesEnabled">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>192.168.0.104</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="QHboxLayout_conn">
              <item>
               <widget class="QPushButton" name="pushButton_conn">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="text">
                 <string>连  接</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_disconn">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="text">
                 <string>断开连接</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_state">
              <property name="text">
               <string>当前连接状态：</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_ipstatus">
              <property name="text">
               <string>Unknown</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Ctrl">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="title">
          <string>控制区域</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_car_light">
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="text">
               <string>车     灯：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_led">
              <property name="text">
               <string>开/关</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_turn_light">
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="text">
               <string>转 向 灯：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_left">
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="text">
               <string>左转</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_off">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="text">
               <string>关闭</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_right">
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="text">
               <string>右转</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_Velocity">
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="text">
               <string>当前车速：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdNumber_Velocity">
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="digitCount">
               <number>3</number>
              </property>
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
              <property name="value" stdset="0">
               <double>0.000000000000000</double>
              </property>
              <property name="intValue" stdset="0">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_degree">
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="text">
               <string>转弯角度：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdNumber_TurnDegree">
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="digitCount">
               <number>3</number>
              </property>
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
              <property name="value" stdset="0">
               <double>0.000000000000000</double>
              </property>
              <property name="intValue" stdset="0">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_img">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="title">
        <string>显示区域</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="Layout_img"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="imgFile.qrc"/>
 </resources>
 <connections/>
</ui>
